sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateTargetedSequencingDao.java

```diff
@@ -0,0 +1,77 @@
+package uk.ac.bbsrc.tgac.miso.persistence.impl;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.criterion.Projections;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import uk.ac.bbsrc.tgac.miso.core.data.impl.TargetedSequencing;
+import uk.ac.bbsrc.tgac.miso.core.store.TargetedSequencingStore;
+
+@Repository
+@Transactional(rollbackFor = Exception.class)
+public class HibernateTargetedSequencingDao implements TargetedSequencingStore {
+
+  protected static final Logger log = LoggerFactory.getLogger(HibernateTargetedSequencingDao.class);
+
+  @Autowired
+  private SessionFactory sessionFactory;
+
+  private Session currentSession() {
+    return getSessionFactory().getCurrentSession();
+  }
+
+  @Override
+  public long save(TargetedSequencing ts) throws IOException {
```

Comment:
Save method is never used, and with our policy to add via migrations, it should probably be removed